server/netty-server/src/main/scala/sttp/tapir/server/netty/internal/NettyServerHandler.scala

```diff
@@ -85,7 +85,9 @@ class NettyServerHandler[F[_]](
             pendingResponses.length
           )
         }
-        pendingResponses.foreach(_.apply())
+        while(pendingResponses.nonEmpty) {
+          pendingResponses.dequeue()()
```

Comment:
I know it's a nit-pick, but maybe `pendingResponses.dequeue().apply()` would be more readable, as the double `()()` look a bit like a typo?

Reply:
good point. I've just updated it